src/stm.py

```diff
@@ -107,12 +97,9 @@ def _cbf_psd(volume, fps, max_freq = 20):
     N = nextpow2(volume.shape[0])
     f, Pxx = signal.periodogram(volume, fs = fps, nfft = N,
         return_onesided = True, axis = 0)
-    max_freq_indices = Pxx.argmax(axis = 0)
-    heatmap = f[max_freq_indices]
-    heatmap[heatmap > max_freq] = max_freq
-    return heatmap
+    return get_best_volumes(Pxx, f, max_freq, sequence)
 
-def _cbf_welch(volume, fps, max_freq = 20):
+def _cbf_welch(volume, fps, max_freq = 10, sequence = 5):
```

Comment:
Keep `max_freq` default value the same across all functions (others are all 20, not 10).